hackbot.js

```diff
@@ -33,8 +33,19 @@ bot.on('message', msg => {
 if (process.env.NODE_ENV === 'production') {
   bot.on('guildMemberAdd', (member) => {
     var { username } = member.user;
-    console.log(`New User '${username}' has joined '${member.guild.name}'` );
-    member.guild.defaultChannel.sendMessage(`'${username}' has joined this server`);
+    console.log(`New User '${username}' has joined '${member.guild.name}'`);
+      
+    const general = member.guild.defaultChannel
+      || member.guild.channels.find("name", process.env.DEFAULT_CHANNEL);
+      
+    if (general) {
+      general.sendMessage(`'${username}' has joined this server`);
+    }
+    else {
+      console.log("This server has no defaultChannel property. (It might be too new.) " +
+                  "If you are running the bot locally, please specify the default " + 
+                  "channel in .env.");
+    }
   });
```

Comment:
@rukokarasu . I think it is a relevant changes especially for people who just got a server up and running. The issue I am seeing is the formatting of the code. Can you please go through the airbnb style guild that is listed in the readme and fix any issues that you may see. The two things that stuck out to me are the if else statement and the concatenating of the string in the console log in the body of the else. There may be others that I am not seeing so be picky. Fix it up then resubmit the pull request and I or someone else will push the changes. If you could also do the pull request to the dev branch that would be awesome. Thanks